### PR TITLE
Type consistency: Use uintptr_t / intptr_t instead of uintptr / intptr

### PR DESCRIPTION
**Pull request by obastemur**
Fixes https://github.com/Microsoft/ChakraCore/issues/168

Replaces all the instances of `intptr`/`uintptr` to `intptr_t`/`uintptr_t`

### CODE REVIEW

**Comment by dilijev** [comment]
@dotnet-bot test EOL Check please

File: lib/Runtime/RuntimeCommon.h [hunk 1]
```diff
@@ -128,20 +128,20 @@ namespace Js
...
-    const uintptr AtomTag           = 0x1;
+    const uintptr_t AtomTag           = 0x1;
     const int32 AtomTag_Multiply    = 1;
     const int32 AtomTag_Pair        = 0x00010001;  // Pair of tags
 #else
-    const uintptr AtomTag_IntPtr     = 0x1;
+    const uintptr_t AtomTag_IntPtr     = 0x1;
     const int32 AtomTag_Int32        = 0x1;    // lower 32-bits of a tagged integer
-    const uintptr AtomTag            = 0x1;
+    const uintptr_t AtomTag            = 0x1;
```
Comment (dilijev) [inline lib/Runtime/RuntimeCommon.h]: Could you fix the alignment of the = in all of the above?

Comment (obastemur) [inline lib/Runtime/RuntimeCommon.h]: @dilijev 
> Could you fix the alignment of the = in all of the above?

Done. Updated.

**Comment by dilijev** [comment]
LGTM.

@jianchun can you also review?

**Comment by obastemur** [comment]
ping!

**Comment by jianchun** [comment]
LGTM too.

**Pull request merged**